etc/kayobe/kolla-image-tags.yml

```diff
@@ -12,9 +12,12 @@ kolla_image_tags:
   horizon:
     rocky-9: 2024.1-rocky-9-20250203T171853
     ubuntu-jammy: 2024.1-ubuntu-jammy-20250203T171853
-  ironic_prometheus_exporter:
-    rocky-9: 2024.1-rocky-9-20250124T081816
-    ubuntu-jammy: 2024.1-ubuntu-jammy-20250124T081816
+  ironic:
+    rocky-9: 2024.1-rocky-9-20250213T110505
+    ubuntu-jammy: 2024.1-ubuntu-jammy-20250213T110505
+  dnsmasq_tag:
```

Comment:
I _think_ this should be ironic_dnsmasq. Either that or just dnsmasq, I can't quite remember

Reply:
Looks like you're right, updated

Reply:
Fixed this properly now, should be good to go